CONTRIBUTING.md

```diff
@@ -67,20 +65,52 @@ All pull requests must pass standard checks:
     * Circleci unit test run (Pytest)
 * Circleci deployment testing (if applicable)
 
-Everything should be runnable in your dev environment, all necessary configuration files are provided
-in VCS. for pylint you may need to manually point it at `pylint.ini`
+Everything should be executable in your development environment, all necessary configuration files are provided
+in VCS. for pylint you may need to manually point at `pylint.ini`
 ## Hygiene
 
 **Please do not squash your commits prior to your PR.**  If this is necessary, it will be brought up during the review process.
 
-Your PR should have sensible commits and messages.
+Your PR should have sensible commits and messages.  Please see the **Pull Requests Naming & Commenting** section below.
 
-All new features are required to contain tests sufficient for newly added code to be evaluated without reducing coverage.
+New features are required to contain tests sufficient for newly added code to be evaluated without reducing coverage.
 
 After branching, you should immediately use `git push -u origin feature/my-awesome-feature` to make that the default upstream ref.
 
 Before opening a PR, rebase onto develop so your PR can be merged fast-forward only, without merge commits.
-The easiest way to do that is to make the main repo a remote using `git remote add upstream git@github.com:fuelrats/pipsqueak` and then running `git pull --rebase upstream develop`.
+The easiest way to do that is to make the main repository a remote using `git remote add upstream git@github.com:fuelrats/pipsqueak` and then running `git pull --rebase upstream develop`.
+
+## Naming & Commenting Standards
+### Pull Requests Naming
+If working from a registered issue, Include the issue name enclosed in brackets in the title of your Pull Request, ie `[SPARK-99] Update to CONTRIBUTING.md`
+
+Otherwise, use the type of request.
+``[Doc] Update TOS to revoke snickers delivery``
+``[Fix] Mechaclient.py should return None after valid on_message event``
+
+You may be asked to rename your Pull Request before peer review can begin if the name is not specific enough or not within standard.
+
+### Peer Review Tagging System
+During a peer review, reviewers will add a tag to the comment to help the author understand what type of action is required to receive an approving review.
+
+| Tag | Explanation |
+| --- | --- |
+[Blocking]|This issue must be fixed before approval.
+[Consensus]|Discussion is required.  Blocked until consensus has been reached by the author and reviewer.
+[Discussion]|Discussion is required, but this is not blocking an approval.
+[Incomplete]|Incomplete code and/or Pull Request.
+[Kudos]|The reviewer liked this, specifically. Go you.
+[Nitpick]|Same as suggestion.  Not blocking, and at authors discretion to change.
```

Comment:
[Discussion]
```
[Nitpick]|Same as suggestion.  Not blocking, and at authors discretion to change.
```
In what way is this different to [Suggestion] and if it isn't, why have it at all?

Reply:
"This is just a nitpick, but..." has been a really common occurrence in recent peer reviews.  Having this as a tag is just an easy way to say "This _may_ be overly picky, but ..."

TLDR, its an easy way to open the subject regarding something that may be considered pedantic, but still merits attention without going into a full blown [discussion].  Overkill, perhaps, but using it is entirely optional in lieu of a more befitting tag.